src/dsp_tools/commands/xmllib/models/file_values.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import annotations
+
+import warnings
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Any
+from typing import Protocol
+
+from lxml import etree
+
+from dsp_tools.commands.xmllib.value_checkers import is_string
+from dsp_tools.models.custom_warnings import DspToolsUserWarning
+from dsp_tools.utils.uri_util import is_iiif_uri
+
+XML_NAMESPACE_MAP = {None: "https://dasch.swiss/schema", "xsi": "http://www.w3.org/2001/XMLSchema-instance"}
+DASCH_SCHEMA = "{https://dasch.swiss/schema}"
+
+
+class AbstractFileValue(Protocol):
+    value: str | Path
+    permissions: str | None
+    comment: str | None = None
+
+    def serialise(self) -> etree._Element:
+        raise NotImplementedError
+
+
+@dataclass
+class FileValue(AbstractFileValue):
+    value: str | Path
+    permissions: str | None = "prop-default"
+    comment: str | None = None
+    resource_id: str | None = None
+
+    def __post_init__(self) -> None:
+        if not is_string(self.value):
```

Comment:
Long-term I would like to check if it is an actual path in case of a file value or a iiif uri... I am waiting for this, because the checks should be cleared with RDU this is somewhat of a place-holder...